service/http/topic.go

```diff
@@ -226,7 +249,7 @@ func (s *Server) AddTopicEventHandler(sub *common.Subscription, fn common.TopicE
 	// Route is only required for HTTP but should be specified for the
 	// app protocol to be interchangeable.
 	if sub.Route == "" {
-		return errors.New("handler route name")
+		return errors.New("missing route for this subscription")
```

Comment:
```suggestion
		return errors.New("missing route name")
```
nit: extraneous wording